### ISSUE TITLE
Property setter detection - detect upper case

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#89]
Example:

```
NSMutableURLRequest *request = ...;
[request setHTTPMethod:@"POST"];
```

Result:
```
let request = ...
request.hTTPMethod = "POST"
```

Should be:
```
let request = ...
request.HTTPMethod = "POST"
```

Before lowercasing the first letter, see if the next one is uppercase as well, in which case, you shouldn't be lowercasing the first letter.

**Comment by drkameleon** [comment]
Ooops. Great catch - hadn't even noticed this one...

**Comment by drkameleon** [comment]
Fixed as of the upcoming 2.0.

---

**Example:**
```objective-c
@implementation aClass

- (void)main {
    [request setHTTPMethod:@"POST"];
    [request setAnotherVariable:@"someVal"];
}

@end
```

**Output:**
```swift
class aClass {

	func main() {
		request.HTTPMethod = "POST"
		request.anotherVariable = "someVal"
	
	}
	
}
```
